keycert/cmc/src/main/java/se/swedenconnect/signservice/certificate/cmc/config/CMCKeyAndCertificateHandlerConfiguration.java

```diff
@@ -69,6 +69,20 @@ public class CMCKeyAndCertificateHandlerConfiguration extends AbstractKeyAndCert
   @Setter
   private RemoteCaInformation remoteCaInfo;
 
+  /**
```

Comment:
I suggest making a HttpProxy object instead. As it is now we can't configure user name or password (should it be needed).

Reply:
Compare with how we solved it for SAML metadata.

Reply:
Comments should be resolved in last update